apps/daimo-mobile/src/model/account.ts

```diff
@@ -25,6 +25,16 @@ import { cacheEAccounts } from "../logic/addr";
 export const defaultEnclaveKeyName =
   process.env.DAIMO_APP_VARIANT === "dev" ? "daimo-dev-12" : "daimo-12";
 
+/**
+ * Poor man's device attestation key name.
+ * Fixed key created once and never deleted -- used as an alternate to
+ * device attestations.
```

Comment:
nit: maybe `deviceApiKeyName` since nothing is being attested

Reply:
sure done